Gruntfile.js

```diff
@@ -451,7 +456,8 @@ module.exports = function(grunt) {
     //'yuidoc:prod', // already done by lint-no-fix
     'build',
     'connect',
-    'mocha',
+    'mochaChrome',
+    //'mocha',
```

Comment:
I'm a little confused about the difference between mocha and mochaChrome, can you explain this? And then if we don't need the mocha call here anymore, let's remove it completely. If we're leaving it as a commented out line, let's add a comment explaining what swapping this in would do.